site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -165,6 +165,12 @@
             <label class="inline-block" for="pinThread">{{ "Pin Thread" }}</label>
             <input type="checkbox" class="pinThread-checkbox" name="pinThread" id="pinThread" value="pinThread" data-ays-ignore="true"/>
         {% endif %}
+        {% if show_expiration and core.getUser().accessFullGrading() %}
+            <div class = "expiration inline-block" style="display: none;">
+              <label class="inline-block" for="expirationDate">{{ "Expiration Date" }}</label>
+              <input type="date" class="date_picker" name="expirationDate" id="expirationDate"/>
```

Comment:
Would it be better to have the value of this textbox default to 7 days in the future? That way the instructor knows that the default is 7 instead of it just setting it to 7 days when the box is empty.

Reply:
Fixed